shell_assignement.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# create a variable to hold the input
+read -p "Enter your name: " userInput
```

Comment:
Hey @ankitsaxena13579  why you have not added "Please" in your message

Reply:
Added please :-|